documentation/src/docs/pravega-concepts.md

```diff
@@ -11,429 +11,312 @@ You may obtain a copy of the License at
 
-For more details on working with State Synchronizers, see [Working with Pravega:
+For more details on working with State Synchronizers, please see [Working with Pravega:
 State Synchronizer](state-synchronizer.md).
 
 ## Architecture
 
-The following figure depicts the components deployed by Pravega:
+The following figure depicts the components deployed by Pravega
 
 ![pravega high level architecture](img/pravega.arch.new.png)
 
 Pravega is deployed as a distributed system – a cluster of servers and storage
-coordinated to run Pravega called a "Pravega cluster".  
+coordinated to run Pravega called a **Pravega cluster**.  
 
-Pravega presents a software-defined storage (SDS) architecture formed by Controller instances 
-(control plane) and Pravega Servers (data plane).The set of Pravega Servers is known collectively as the Segment Store. 
+Pravega presents a software-defined storage (SDS) architecture formed by **Controller** instances
+(_control plane_) and Pravega Servers (_data plane_).The set of Pravega Servers is collectively known as the **Segment Store**. 
 
-The set of Controller instances make up the control plane of Pravega, providing
-functionality to create, update and delete Streams, retrieve information about
-Streams, monitor the health of the Pravega cluster, gather metrics etc.  There
-are usually multiple (recommended at least 3) Controller instances running in a
+The set of Controller instances together forms the control plane of Pravega, providing
+functionality to _create, update_ and _delete_ _Streams_. Further, it extends the functionality to retrieve information about the Streams, monitor the health of the Pravega cluster, gather metrics, etc.,. A minimum requirement of three or multiple Controller instances are allowed to be running in a
 cluster for high availability.  
 
-The [Segment Store](segment-store-service.md) implements the Pravega data plane.
- Pravega Servers provide the API to read and write data in Streams.  Data
-storage is comprised of two tiers: Tier 1 Storage, which provides short term,
-low-latency, data storage,  guaranteeing the durability of data written to
-Streams and Tier 2 Storage providing longer term storage of Stream data.
- Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
-Tier 1 Storage and uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud
-Storage (ECS) to implement Tier 2 Storage.  Tier 1 Storage typically runs within
-the Pravega cluster.  Tier 2 Storage is normally deployed outside the Pravega
-cluster.
-
-Tiering storage is important to deliver the combination of fast access to Stream
-data but also allow Streams to store a vast amount of data.  Tier 1 storage
-persists the most recently written Stream data.  As data in Tier 1 Storage ages,
+The [Segment Store](Segment-store-service.md) implements the Pravega data plane.
+Pravega servers provide the API to read and write data in Streams. Data storage is comprised of two tiers:
```

Comment:
`Pravega servers` is a concept in terminology, so let's write it in caps as well.

Reply:
Accepted Suggesstion